axlearn/common/conformer.py

```diff
@@ -253,6 +259,11 @@ def __init__(self, cfg: Config, *, parent: Module):
                 f"cfg.right_context must be greater or equal to 0, get {cfg.right_context}."
             )
 
+        if cfg.layer_order is not None:
+            supperted_layer_order = ["lconv_before_ff", "lconv_before_mhsa", "mhsa_before_lconv"]
```

Comment:
```suggestion
            supported_layer_order = ["lconv_before_ff", "lconv_before_mhsa", "mhsa_before_lconv"]
```
This doesn't look fixed?

Reply:
Sorry. Missing this one. Fixed now.